vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ModuleLinkManager.java

```diff
@@ -104,4 +111,10 @@ protected void deleteLinkRepo(IModuleLink link) {
         moduleLinkRepo.delete((ModuleLink) link);
     }
 
+    @Override
+    public HashSet<ISpace> findSpaceLinksFromModuleId(String moduleId) {
```

Comment:
this method returns a list of spaces, why is is called "findSpaceLinks..."?

Reply:
fixed